.github/workflows/upsert-study.yml

```diff
@@ -0,0 +1,85 @@
+on:
+  workflow_dispatch:
+    inputs:
+      name:
+        description: "Study name"
+        required: true
+        type: string
+      enable_feature:
+        description: "Feature name to enable"
+        required: true
+        type: string
+      probability_enabled:
+        description: "Percentage number to enable. ex: 25"
+        required: true
+        default: "25"
+        type: string
+      channel:
+        description: "Channels to enable on, in comma separated value format"
+        required: true
+        default: "RELEASE,BETA,NIGHTLY"
+        type: string
+      platform:
+        description: "Platforms to enable on, in comma separated value format"
+        required: true
+        default: "WINDOWS,MAC,LINUX,ANDROID"
```

Comment:
Sort?

Reply:
Don't think we should here. The json file contains this particular order everywhere, and I think it's better to add the new ones in the same order